src/server/graphql/models/Organization/addBilling/addBilling.js

```diff
@@ -107,6 +126,16 @@ export default {
             })
             .delete();
         });
+      if (segmentIo) {
```

Comment:
dupe of line 56?

think it's worthwhile to mock segmentIo if `SEGMENT_WRITE_KEY` doesn't exist? then we wouldn't have to wrap every call in a conditional